src/passes/Print.cpp

```diff
@@ -1512,8 +1511,7 @@ std::ostream& WasmPrinter::printStackIR(StackIR* ir, std::ostream& o, Function*
       o << ' ';
     }
   };
-  for (Index i = 0; i < (*ir).size(); i++) {
-    auto* inst = (*ir)[i];
+  for (auto inst : (*ir)) {
```

Comment:
Looks like the brackets are no longer needed?  But maybe you want to keep this as fully automated CL?